packages/gulp-tasks/package-lock.json

```diff
@@ -1155,15 +1155,6 @@
         }
       }
     },
-    "@ima/helpers": {
```

Comment:
This doesn't seems right, afaik the gulp tasks till have `@ima/helpers` in package.json devDeps so it should probably be in a lock too.